### PR TITLE
css-flexbox - Vorleak Yek

### PR DESCRIPTION
**Pull request by vorleakyek**
image
image
image
image

### CODE REVIEW

File: css-flexbox/css-flexbox-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is the default `flex-direction` of a `flex` container?
+
+- What is the default `flex-wrap` of a `flex` container?
+
+- What axis do the `justify-content` properties work on?
+
+- What axis do the `align-items` properties work on?
+
```
Comment (kurtyazdizadeh) [inline css-flexbox/css-flexbox-quiz-notes.md]: please don't forget to answer the quiz questions :)

**Review by kurtyazdizadeh — CHANGES_REQUESTED** [review]
Please answer the quiz questions - otherwise it looks good. I'll approve once the quiz questions are answered.

File: css-flexbox/css-flexbox-quiz-notes.md (hunk 1 above)
Comment (vorleakyek) [reply css-flexbox/css-flexbox-quiz-notes.md]: Okay, I just made the changes to include the answer to the quiz questions. Thanks!

**Review by kurtyazdizadeh — APPROVED** [review]
Approved 👍

**Pull request merged**